frontend/public/components/app-topology/app-topology-pod-donut.jsx

```diff
@@ -0,0 +1,132 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'patternfly/dist/js/patternfly-settings';
+
+const { patternfly } = window;
+import { Donut } from '../graphs';
+import { PodUtils } from '../../module/utils/pod-utils';
+
+const POD_COLORS = {
+  Empty: patternfly.pfPaletteColors.black100,
+  Running: patternfly.pfPaletteColors.lightBlue,
+  "Not Ready": patternfly.pfPaletteColors.lightBlue100,
+  Warning: patternfly.pfPaletteColors.orange300,
+  Error: patternfly.pfPaletteColors.red200,
+
+  Pulling: patternfly.pfPaletteColors.black300,
+  ContainerCreating: patternfly.pfPaletteColors.black300,
+
+  Pending: patternfly.pfPaletteColors.black200,
+  Succeeded: patternfly.pfPaletteColors.green,
+  Terminating: patternfly.pfPaletteColors.blue500,
+  Unknown: patternfly.pfPaletteColors.gold200
+};
+
+export class AppTopologyPodDonut extends React.Component {
```

Comment:
We already have a [donut chart](https://github.com/openshift/console/blob/master/frontend/public/components/graphs/donut.jsx).

Reply:
Yes, this component utilizes that.